resources/tests/static_cpu_templates/aarch64_remove_ssbs.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
what's the purpose of this template? It can't be used without a patched kernel either, can it?

Reply:
It just disabled ssbs feature. It works on both G2 and G3.
None of arm templates can be used without patched kernel. There is a separate pipeline for them with instances with patched kernel.

Reply:
I think if the ssbs template can't be used without the patch, I can't really see a value in having it.

Reply:
It seems we figured this out